### DISCUSSION TITLE
Why is my cursor locked on the right side of the screen?

### DISCUSSION THREAD

**Discussion by Porthos-0x45** [ppy/osu#19614] [Q&A] [answered]
I wanted to change my sensitivity thus enabling the high precious mouse thinking it's something like raw input, but It just locked my cursor on the side of the window (I tried using the window mode). 
I use OpenTabletDriver and yes I have the windows ink disabled.

I really want to change the sensitivity because it's really important in my play style and I think in many others too.

(Found a discussion similar to this one but that was 2021 and it wasn't the same issue I got here)

![osu_2022-08-07_01-54-43](https://user-images.githubusercontent.com/99044000/183269538-b282c4b5-2699-4280-8ca8-0249069f5b65.jpg)
![osu_2022-08-07_01-55-30](https://user-images.githubusercontent.com/99044000/183269539-5436f933-0548-41a4-a03c-90658dee9646.jpg)
![osu_2022-08-07_01-55-31](https://user-images.githubusercontent.com/99044000/183269540-8cd30a26-7193-4a17-b819-244856532849.jpg)

**Comment by peppy** [comment, marked as answer]
If you are using an external tablet driver, please adjust your area / sensitivity there rather than in-game. There is actually no way we can correctly adjust it in game, because you have external software sending absolute positioning data.

**Reply by Porthos-0x45** [reply]
Oh ok, thank you for your quick response.
